aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -53,6 +53,8 @@ def define(cls, spec):
         spec.input('code', valid_type=orm.Code, help='The Code to use for this job.')
         spec.input('metadata.dry_run', valid_type=bool, default=False,
             help='When set to True will prepare the calculation job for submission but not actually launch it.')
+        spec.input('metadata.computer', valid_type=orm.Computer, required=False,
+            help='Set the computer to be used by the calculation')
```

Comment:
Perhaps mention here that this is for local codes only, something like
```suggestion
            help='Set the computer to be used by the calculation (for "local" codes only)')
```